plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -209,37 +272,52 @@ proc string mayaUsdTranslatorExport_AppendConvertMaterialsTo(string $currentOpti
 }
 
 global proc mayaUsdTranslatorExport_EnableAllControls() {
+    global string $mayaUsdTranslatorExport_SectionNames[];
+    string $sectionNames[] = $mayaUsdTranslatorExport_SectionNames;
+    
     // Restore all controls to fully interactive:
-    checkBoxGrp -e -en 1 exportUVsCheckBox;
-    checkBoxGrp -e -en 1 exportBlendShapesCheckBox;
-    checkBoxGrp -e -en 1 exportDisplayColorCheckBox;
-    checkBoxGrp -e -en 1 exportColorSetsCheckBox;
-    checkBoxGrp -e -en 1 animationCheckBox;
-    checkBoxGrp -e -en 1 eulerFilterCheckBox;
-    checkBoxGrp -e -en 1 staticSingleSampleCheckBox;
-    checkBoxGrp -e -en 1 exportVisibilityCheckBox;
-    checkBoxGrp -e -en 1 mergeTransformAndShapeCheckBox;
-    checkBoxGrp -e -en 1 includeNamespacesCheckBox;
-
-    optionMenuGrp -e -en 1 skelsPopup;
-    optionMenuGrp -e -en 1 skinClustersPopup;
-    optionMenuGrp -e -en 1 defaultMeshSchemePopup;
-    optionMenuGrp -e -en 1 defaultUSDFormatPopup;
-    optionMenuGrp -e -en 1 exportInstancesPopup;
-    optionMenuGrp -e -en 1 jobContextPopup;
-
-    intFieldGrp -e -en1 1 -en2 1 frameRangeFields;
-
-    floatFieldGrp -e -en1 1 frameStrideField;
-
-    textFieldGrp -e -en 1 frameSampleField;
-    textFieldGrp -e -en 1 parentScopeField;
+    if (stringArrayContains("geometry", $sectionNames)) {

```

Comment:
Are these strings (like "geometry") from a string resource? If yes, then you need to compare against the string from resource.

Reply:
No. They are internal strings to communicate what part of the UI to show or not. They are not localized.

Reply:
okay thanks.